percona/pgbackrest/pgbackrest.go

```diff
@@ -56,7 +58,12 @@ func GetInfo(ctx context.Context, pod *corev1.Pod, repoName string) (InfoOutput,
 	}
 
 	for _, elem := range out {
-		if elem.Status.Code != 0 {
+		switch elem.Status.Code {
+		case 2:
+			return InfoOutput{}, ErrNoValidBackups
+		case 0:
+			continue
```

Comment:
it'd be nice to put some comments explaining what do these statuses mean

Reply:
https://github.com/percona/percona-postgresql-operator/pull/763/commits/eacf467a558fa284ca458a68166766367843cedd